keyboards/meletrix/zoom75/readme.md

```diff
@@ -0,0 +1,27 @@
+# zoom75
+
+More Info at [meletrix](https://meletrix.com/)
```

Comment:
Could you please clarify here how this sits/interacts with the bluetooth hardware, to avoid inevitable confusion.

Reply:
I don't know either, I am just a college student, I write single mode version.😂 Bluetooth version solutions should be provided by third-party contractors. And they provided screen solutions too. I'm also curious about how to realize Bluetooth full-key no punching.
🤨